docs/api-v2-import.json

```diff
@@ -0,0 +1,520 @@
+{
+    "openapi": "3.0.0",
+    "info": {
+      "version": "2.0.0",
+      "title": "Scarf data import api",
+      "description": "This API specification is for importing event data into Scarf"
+    },
+    "servers": [
+      {
+        "url": "http://api.scarf.sh"
+      }
+    ],
+    "paths": {
+      "/v2/imports/{owner}/{ticket}/log": {
+        "parameters": [
+          {
+            "name": "owner",
+            "in": "path",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "Owner of the event import.\n"
+          },
+          {
+            "name": "ticket",
```

Comment:
Why `ticket` as a name here? That name seems like it could be for a multitude of implementations (support ticket?, import task ticket?, aggregate ticket?). Doesn't seem indicative of its purpose as an import ID. May need to choose a different name that makes more sense for what it's being used for.

Reply:
I did remove any outdated mentions.